x/pki/x509/x509.go

```diff
@@ -101,14 +102,18 @@ func FormatOID(header, oldKey, newKey string) string {
 	return strings.Join(subjectValues, ",")
 }
 
+func StringtoBase64(subject string) string {
```

Comment:
Typo. Should be `StringToBase64`.
I believe it would be better to call it `ToBase64String`

Reply:
Done! ✅